examples/40-access-entries.yaml

```diff
@@ -6,7 +6,7 @@ kind: ClusterConfig
 metadata:
   name: access-entries-cluster
   region: us-west-2
-  version: '1.25'
```

Comment:
Did this example work before? or it's not being tested at all?

Reply:
Yeah it worked with non `STANDARD` types. I just updated to show the new fields in use